### DISCUSSION TITLE
Embedded Dasboard option is not coming even after setting EMBEDDED_SUPERSET=true in config.py

### DISCUSSION THREAD

**Discussion by sandeeppennant** [apache/superset#23989] [General]
Even after setting EMBEDDED_SUPERSET=true under DEFAULT_FEATURE_FLAGS in config.py the Embedded Dasboard.
![Embedded_DashBoard](https://github.com/apache/superset/assets/133017546/49cebd18-e26a-47bf-bf3f-1885b5ce6b05)

**Comment by lemaadi** [comment]
Hi there,

I have the same issue. It is working locally but i am unable to see the `Embed dashboard` option after deploying the app to aws ecs fargate instance

I am using `apache/superset:latest` docker image
I added `FEATURE_FLAGS = {"ALERT_REPORTS": True,  "EMBEDDED_SUPERSET": True}` to `superset_config.py` file
I am using version `0.0.0dev`

Your suggestions/help is much appreciated

**Comment by lucidprojects** [comment]
I have the same issue.  I have the feature flag added to my config but it is not an option in the dropdown.

**Comment by tanmaykansara** [comment]
Were any of you able to resolve this? I am also not seeing the embedd dashboard link. Ty

**Reply by lucidprojects** [reply]
My issue was my config changes were not actually getting applied. It was using the default config.py file, not my edited superset_config.py

I was able to resolve this issue be explicitly mounting the volume.  

```  
docker run -d -p 8088:8088 \
  -v ./docker/pythonpath/superset_config.py:/app/superset_config.py \
  --name superset apache/superset
  ```

Once I had that correctly mounted I was able to see the drop down option.

Unfortunately, I ran into the role API endpoint failure bug talked about [here](https://github.com/apache/superset/issues/19525). Although a recent post on that thread looks like there may be a solution, I haven't had a chance to try it.

**Reply by tanmaykansara** [reply]
Ty for sharing !!

**Comment by senaranya** [comment]
Create a `docker/.env-local` and add following in it:
```
SUPERSET_FEATURE_EMBEDDED_SUPERSET=true
```
It's working for me without needing any other changes

**Reply by jackgray** [reply]
This is the only thing that worked after a full day spent reading documentation and all the tutorials I could find on the subject. I wonder why none of the official documentation has it right..

Thank you @senaranya -- you saved me from going mad!
